concourse/tasks/update-ecs-service.yml

```diff
@@ -30,7 +30,7 @@ run:
 
       echo "Updating $APPLICATION service..."
 
-      task_definition_arn=$(cat terraform-outputs/task_definition_arn)
+      task_definition_arn=$(cat "terraform-outputs/$APPLICATION_task_definition_arn")
```

Comment:
Thanks 👍

Reply:
:neckbeard:  but I think your change to the quotes isn't right.

You _can_ omit quotes on the right hand side of an assignment ([stackexchange answer](https://unix.stackexchange.com/questions/68694/when-is-double-quoting-necessary)), but you are subject to word splitting in the argument to `cat`.

So if `APPLICATION` is set to `more than one word` here, you'll get:

```
task_definition_arn="$(cat terraform-outputs/more than one word_task_definition_arn)"
```

which isn't what you want. I did a quick check in shellcheck, just to be sure.

This would be fine if you wanted to err on the side of always quoting things (which is reasonable):

```
task_definition_arn="$(cat "terraform-outputs/${APPLICATION}_task_definition_arn")"
```

Definitely agree the braces are an improvement though.

Reply:
We could consider running the paas' teams [pipecleaner](https://github.com/alphagov/paas-cf/tree/master/tools/pipecleaner) on our pipelines. It's very clever.

Reply:
Oops - yes you're right! (I forgot that assignment is special in that regard.) We should just run this stuff through shellcheck instead of relying on fallible reviewers to do the work of the linter 😅

Reply:
It might still be a good idea to use the quotes around the whole thing for defensive reasons (i.e. in case it gets refactored later such that it's no longer the right-hand side of an assignment) - I don't know what shellcheck's defaults have to say about it though.

Reply:
Shellcheck is fine with having both sets of quotes (it's quite good in that it mostly only nags you about things that might be bugs, rather than encouraging you to play shell golf ⛳ ).

PR to fix: https://github.com/alphagov/govuk-infrastructure/pull/85